packages/firestore/test/integration/browser/webchannel.test.ts

```diff
@@ -32,6 +32,54 @@ const describeFn =
       xdescribe;
 
 describeFn('WebChannel', () => {
+  it.only('receives error messages', done => {
+    const projectId = DEFAULT_PROJECT_ID;
+    const info = getDefaultDatabaseInfo();
+    const conn = new WebChannelConnection(info);
+    const stream = conn.openStream<api.ListenRequest, api.ListenResponse>(
+      'Listen',
+      null
+    );
+
+    // Test data
+    const payload = {
+      database: 'projects/' + projectId + '/databases/(default)',
+      addTarget: {
+        query: {
+          parent: 'projects/' + projectId + '/databases/(default)',
+          structuredQuery: {
+            from: [{ collectionId: 'WHAT_COLLECTION_ID_SHOULD_THIS_USE' }]
+          }
+        }
+      }
+    };
+
+    // Once the stream is open, send an "add_target" request.
+    stream.onOpen(() => {
+      stream.send(payload);
+    });
+
+    stream.onMessage(() => {
+      // Shut down the stream since we don't care about processing any more messages
+      stream.close();
+    });
+
+    // Ensure Time to first byte was raised 
+    let timeToFirstByteData:{ type: number, timeToFirstByteMs: number } |null = null;
```

Comment:
```suggestion
    let timeToFirstByteData:{ type: number, timeToFirstByteMs: number };
```

For brevity (if it compiles - I am not sure right now).

Reply:
Originally it didn't compile which is why I have it there.